.travis.yml

```diff
@@ -6,8 +6,12 @@ before_script:
 # gdb is used by some dmd tests
 - sudo apt-get install gdb
 script:
+# install rund
+- git clone --depth=1 --branch=master https://github.com/marler8997/rund.git ../rund
+- dmd -I../rund/src -run ../rund/make.d build
+- sudo cp ../rund/bin/rund /bin/rund
```

Comment:
Btw using Travis in a docker instance without sudo will result in faster spinup times. Though, alternatively CircleCi does a good job too.

Reply:
https://github.com/marler8997/dc/issues/48